HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -310,6 +310,47 @@ code::
 "mississippi".split("i").postcs;
 ::
 
+method::insertAll
+Return a new collection of the same kind, with all elements of another collection or object inserted, starting from a given index.
+
+argument::index
+The index at which to start insertion. If too large or too small, append or prepend the object's items.
+
+argument::obj
+The obj to insert (usually a collection, but anything that responds to code::++::).
+
+discussion::
+code::
+a = [1, 2, 3, 4, 5, 6];
+b = [100, 200];
+a.insertAll(2, b); // [ 1, 2, 100, 200, 3, 4, 5, 6 ]
+::
+
+method::mergeItems
+Return a new collection that has all items of the receiver, with elements from another collection inserted where not already present (test is strong::equality::). This is done in a way that preserves the orders of both collections if possible. If the argument collection has inversed order, the order of the receiver is kept.
```

Comment:
This feels somewhat exotic to me... maybe a sentence on application?

also, "Return a new collection that has all items " >> "Return a new collection that contains all items"